spec/libsass-todo-issues/issue_1098/expected.expanded.css

```diff
@@ -0,0 +1,5 @@
+div {
+  opacity: 1\9;
+  width: 500px\9;
+  color: #f00\9\0\;;
```

Comment:
I believe this double `;;` may be an issue with Sass. /cc @mgreter @chriseppstein